libs/klv/src/test/java/org/codice/alliance/libs/klv/DistinctKlvProcessorTest.java

```diff
@@ -13,18 +13,68 @@
  */
 package org.codice.alliance.libs.klv;
 
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
+import com.google.common.collect.ImmutableList;
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.MetacardImpl;
+import java.io.Serializable;
+import java.util.List;
 import org.junit.Test;
 
 public class DistinctKlvProcessorTest {
 
+  public static final String ATTRIBUTE_NAME = "title";
+
+  @Test
+  public void doProcessWithActualValue() {
+    Metacard metacard = setupAndExecuteDoProcess(ImmutableList.of("test-value"));
+
+    assertThat(metacard.getAttribute(ATTRIBUTE_NAME), is(not(nullValue())));
+    assertThat(metacard.getAttribute(ATTRIBUTE_NAME).getValue().toString(), equalTo("test-value"));
+  }
+
   @Test
-  public void testAccept() {
+  public void doProcessWithBlankValues() {
+    Metacard metacard = setupAndExecuteDoProcess(ImmutableList.of("", " "));
+
+    assertThat(metacard.getAttribute(ATTRIBUTE_NAME), is(nullValue()));
+  }
+
+  @Test
+  public void doProcessWithBlankValuesAndActualValue() {
+    Metacard metacard = setupAndExecuteDoProcess(ImmutableList.of("test-value", " "));
+
+    assertThat(metacard.getAttribute(ATTRIBUTE_NAME), is(not(nullValue())));
+    assertThat(metacard.getAttribute(ATTRIBUTE_NAME).getValue().toString(), equalTo("test-value"));
+  }
```

Comment:
❓ This test verifies that the 2nd input value is dropped and the result is a single-valued attribute. How about another test that verifies a 3rd input value is dropped and the correct multi-valued attribute is returned.